backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/audit/data/model/DataExportAudit.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2021 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.audit.data.model;
+
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+import org.adempiere.exceptions.AdempiereException;
+import org.adempiere.util.lang.impl.TableRecordReference;
+
+import javax.annotation.Nullable;
+
+@Value
+@Builder
+public class DataExportAudit
+{
+	@Nullable
+	DataExportAuditId id;
+
+	@NonNull
+	TableRecordReference tableRecordReference;
+
+	@Nullable
+	DataExportAuditId parentId;
+
+	@NonNull
+	public DataExportAuditId getIdNotNull()
```

Comment:
having a nullable ID is usually a sign that you are mixing the Command and Query classes in one class.
So you would need 2 classes:
* DataExportAudit - is the actual model, the id shall NOT be null
* CreateDataExportAuditRequest - the create request used to create the database record.